pages/index.js

```diff
@@ -555,36 +574,56 @@ export default function Home({
             user && (
               <>
                 <StyledAnimationContainer>
-                  <div>
+                  <div style={{width: "40%"}}>
                     <h1 style={{textAlign: "center"}}>
                       Welcome {user.firstName}
                     </h1>
                     <p>Want to see your accounts?</p>
+                    <Lottie
+                      style={{marginTop: "-2rem"}}
+                      options={defaultOptionsArrows}
+                      width={"20%"}
+                      height={"50%"}
+                    />
+                  </div>
+                  <div style={{height: "100%", width: "60%"}}>
+                    <Lottie
+                      options={defaultOptionsWelcome}
+                      width={"100%"}
+                      height={"100%"}
+                    ></Lottie>
```

Comment:
Don't use inline styling for the `<div>`s, use styled components instead.

Reply:
For just 2 properties it's more code to write a new styled component instead of just using inline styling... Can't I leave it like this?